appcues/src/main/res/values/strings.xml

```diff
@@ -38,6 +38,7 @@
     <string name="appcues_debugger_status_check_push_error_no_permission">Error 2: Notification permission required. Tap to enable through settings</string>
```

Comment:
side note, i noticed Error 2 here happens before Error 1 in the check sequence - should we have our errors ordered to match the order of the checks?

Reply:
they were added in the order that i thought of them basically, but if you agree with the order they are being checked  i can rename the strings to match that

Reply:
yeah i was thinking order them to match the checks - since it might be confusing to get Error 2, fix it, then get Error 1 - feel like you are going backwards. minor thing